src/visualizer/sortingvisualizer/Visualizer.js

```diff
@@ -67,6 +67,11 @@ const Visualizer = () => {
     }
   }, [isPlay, speed, dataSize, visualizerAlgorithm, arrayData]);
 
+  /**
+   * Changes the number of "block" or "ovals" for the sorting animation.
+   *
+   * @param val {number} The number of "block" or "ovals" for sorting animation.
```

Comment:
This and all the below jsdoc seem to follow parameter name, @type, which does not follow the style used in jsdoc?

![image](https://user-images.githubusercontent.com/65328119/103145109-c101e180-476f-11eb-8777-4cf67a95d2c8.png)